src/components/Videocard/VideoCard.js

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react";
+import PageContext from "../../providers/Context/PageContext";
+import { Link } from "react-router-dom";
+import * as Styles from "./styles/VideoCardStyles";
+
+const VideoCard = (props) => {
+  const { video, setVideo } = useContext(PageContext);
+  const { setObjVideo } = useContext(PageContext);
+
+  const HandleVideo = () => {
+    const videoAux = { ...video };
+    videoAux.id = props.id;
+    videoAux.title = props.titulo;
+    videoAux.publishedDate = props.fecha;
+    videoAux.description = props.descripcion;
+
+    setVideo(videoAux);
+  };
+
+  const SetData = () => {
+    HandleVideo();
+    setObjVideo(props.objeto);
+  };
+
+  return (
+    <div onClick={SetData}>
```

Comment:
can't you apply the onClick event to Styles.Card instead? Doing that way, the div tag is not necessary.